extensions/wikia/Recirculation/RecirculationController.class.php

```diff
@@ -57,8 +61,12 @@ public function discussions() {
 	}
 
 	public function footer() {
-		global $wgSitename, $wgCityId;
+		global $wgSitename, $wgCityId, $wgLanguageCode;
 
+		if ($wgLanguageCode !== 'en') {
+			//This is temporary to supress MCF for old discussions on non-en wikis;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)